tests/baselines/reference/emitArrowFunction.js

```diff
@@ -0,0 +1,29 @@
+//// [emitArrowFunction.ts]

```

Comment:
It would be really helpful if the baseline stated the target, just like the original test.